### PR TITLE
Change default mediatype for strings

### LINKED ISSUE #14: default mediatype for string inputs to encode
Should be `text/plain;charset=utf8`.  Leave the `default application/octet-stream` for Buffers

### PR DESCRIPTION
**Pull request by hildjj**
When encoding a string, use text/plain;charset=UTF-8

Fixes #14

### CODE REVIEW

File: test/encode.js [hunk 1]
```diff
@@ -31,11 +31,24 @@ describe('encode', function() {
...
   });
 
+  it('creates from buffer with default mediatype', function() {
+    var uri = dataUri.encode(new Buffer('foo', 'utf8'));
+    expect(uri).to.be.a('string');
+    expect(uri).to.equal('data:application/octet-stream;base64,Zm9v');
+  });
+
   it('creates from string', function() {
     // default mediatype
```
Comment (bajtos) [inline test/encode.js]: A forgotten comment? I think it should be removed, because this test is setting an explicit mediatype "text/xml".

**Comment by bajtos** [comment]
LGTM in general, please address the comment above.

File: test/encode.js (hunk 1 above)
Comment (hildjj) [inline test/encode.js]: You mean on line 41, I think.  Wiifix.

**Comment by hildjj** [comment]
@bajtos, this is ready to go, rebased on top of current head and squashed, with your comment fixed.

**Comment by bajtos** [comment]
Landed, thank you!

**Comment by bajtos** [comment]
Can you update the README too please?

**Pull request merged**